deno/rest/v10/mod.ts

```diff
@@ -834,6 +834,14 @@ export const Routes = {
 	guildScheduledEventUsers(guildId: Snowflake, guildScheduledEventId: Snowflake) {
 		return `/guilds/${guildId}/scheduled-events/${guildScheduledEventId}/users` as const;
 	},
+
+	/**
+	 * Route for
+	 * - POST /channels/${channel.id}/threads
+	 */
+	guildForumThreads(channelId: Snowflake) {
```

Comment:
Idk how to feel about this, is this not a route already? if it is, alias it please